Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -308,6 +311,17 @@ internal void RenameActionItem(int index)
             m_ActionsTreeView.GetRootElementForIndex(index)?.Q<InputActionsTreeViewItem>()?.FocusOnRenameTextField();
         }
 
+        private void FinishActionRename(int id)
+        {
+            if (!m_FocusOnRenameActionFinish || id == -1)
+                return;
+            m_ActionsTreeView.ScrollToItemById(id);
+            var treeViewItem = m_ActionsTreeView.GetRootElementForId(id)?.Q<InputActionsTreeViewItem>();
+            treeViewItem?.FocusOnRenameFinish();
```

Comment:
Does it really needs to use this new function ?
the Id here might not be the expected one. If during the rename you click on another element "id" will be the selected id and not the one that was renamed.

Locally it seems to work well with **treeViewItem?.FocusOnRenameFinish();** replaced by **treeViewItem?.Focus();**